viewlog.c

```diff
@@ -40,12 +41,14 @@ extern options_t o;
 void ViewLog(int config)
 {
   TCHAR filename[2*MAX_PATH];
+  TCHAR assocexe[2*MAX_PATH];
+  DWORD assocexe_num;
 
```

Comment:
size of MAX_PATH for files (paths) and 2*MAX_PATH only for the command line which is made of two paths.